### PR TITLE
reef: mgr/dashboard/frontend:Ceph dashboard supports multiple languages

### PR DESCRIPTION
**Pull request by Svelar**
backport tracker: https://tracker.ceph.com/issues/65028

---

backport of https://github.com/ceph/ceph/pull/52617
parent tracker: https://tracker.ceph.com/issues/58653

this backport was staged using ceph-backport.sh version 16.0.0.6848
find the latest version at https://github.com/ceph/ceph/blob/main/src/script/ceph-backport.sh

### CODE REVIEW

**Review by nizamial09 — APPROVED** [review]

**Comment by nizamial09** [comment]
let's wait for the qa to be completed before merging this.

**Comment by Svelar** [comment]
Jenkins build: https://shaman.ceph.com/builds/ceph/wip-svelar-testing-reef-2024-03-26-0716/

**Comment by Svelar** [comment]
https://pulpito.ceph.com/svelar-2024-03-27_06:35:39-rados-wip-svelar-testing-reef-2024-03-26-0716-distro-default-smithi/

Seems like unrelated failed and dead jobs

**Comment by Svelar** [comment]
> https://pulpito.ceph.com/svelar-2024-03-27_06:35:39-rados-wip-svelar-testing-reef-2024-03-26-0716-distro-default-smithi/
> 
> Seems like unrelated failed and dead jobs

Would merge this one if no objection occurs in two days.

**Comment by nizamial09** [comment]
> > https://pulpito.ceph.com/svelar-2024-03-27_06:35:39-rados-wip-svelar-testing-reef-2024-03-26-0716-distro-default-smithi/
> > Seems like unrelated failed and dead jobs
> 
> Would merge this one if no objection occurs in two days.

Lets wait till @yuriw finishes the teuth suite before merging this.

**Comment by Svelar** [comment]
> > > https://pulpito.ceph.com/svelar-2024-03-27_06:35:39-rados-wip-svelar-testing-reef-2024-03-26-0716-distro-default-smithi/
> > > Seems like unrelated failed and dead jobs
> > 
> > 
> > Would merge this one if no objection occurs in two days.
> 
> Lets wait till @yuriw finishes the teuth suite before merging this.

Of course that is fine. But any insufficiency with my teuthology run?

**Comment by nizamial09** [comment]
> But any insufficiency with my teuthology run?

Nope, yours also looks fine but since a run is now scheduled against this PR, its good to wait and see its results as well. Btw you can track that run here: https://tracker.ceph.com/issues/65202

**Comment by Svelar** [comment]
> > But any insufficiency with my teuthology run?
> 
> Nope, yours also looks fine but since a run is now scheduled against this PR, its good to wait and see its results as well. Btw you can track that run here: https://tracker.ceph.com/issues/65202

Sorry for missing the detail, sounds reasonable to wait.

**Comment by Svelar** [comment]
cc. @TomNewChao

**Comment by ljflores** [comment]
Rados approved: https://tracker.ceph.com/projects/rados/wiki/REEF#httpstrackercephcomissues65202

**Pull request merged**
